package.json

```diff
@@ -34,18 +37,39 @@
 	"devDependencies": {
 		"enzyme": "^3.3.0",
 		"enzyme-adapter-react-16": "^1.1.1",
+		"eslint-plugin-react-native": "^3.2.1",
```

Comment:
Do we need the this? We're not using React Native - eslint-plugin-react should probably be a dev dependency instead though